recipes/log.c/all/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(log.c
+	    VERSION 0.1.0
```

Comment:
There are no releases https://github.com/rxi/log.c/releases, please follow the virtual versioning outlined in the docs

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
Fix it